wire/msgtx.go

```diff
@@ -83,6 +83,21 @@ const (
 	// of transaction signatures + LockTime 4 bytes + Expiry 4 bytes + min
 	// input payload + min output payload.
 	minTxPayload = 4 + 1 + 1 + 1 + 4 + 4
+
+	// freeListMaxScriptSize is the size of each buffer in the free list
+	// that	is used for deserializing scripts from the wire before they are
+	// concatenated into a single contiguous buffers.  This value was chosen
+	// because it is slightly more than twice the size of the vast majority
+	// of all "standard" scripts.  Larger scripts are still deserialized
+	// properly as the free list will simply be bypassed for them.
+	freeListMaxScriptSize = 512
+
+	// freeListMaxItems is the number of buffers to keep in the free list
+	// to use for script deserialization.  This value allows up to 100
+	// scripts per transaction being simultaneously deserialized by 125
+	// peers.  Thus, the peak usage of the free list is 12,500 * 512 =
+	// 6,400,000 bytes.
+	freeListMaxItems = 12500
```

Comment:
And of course if the `feeListMaxScriptSize` is tweaked, this and/or the comment would need to be updated too.